gemfile.d/katello.rb

```diff
@@ -1 +1 @@
-gem 'katello', github: 'Katello/katello', branch: 'master'
+gem 'katello', github: 'ofedoren/katello', branch: 'feat-37852-support-rails-7'
```

Comment:
I guess we keep this unmerged until Katello and Foreman get their fixes?

Reply:
Yeah, and it's mostly to keep this PR green.